codegen/src/main/java/software/amazon/awssdk/codegen/poet/rules/EndpointRulesClientTestSpec.java

```diff
@@ -781,8 +783,26 @@ private CodeBlock createMemberValue(MemberModel memberModel, TreeNode valueNode)
         }
 
         if (memberModel.isMap()) {
-            // Not necessary at the moment
-            throw new RuntimeException("Don't know how to create map member.");
+            Iterator<Map.Entry<String, JrsValue>> fieldsIter = ((JrsObject) valueNode).fields();
+            MemberModel keyModel = memberModel.getMapModel().getKeyModel();
+            MemberModel valueModel = memberModel.getMapModel().getValueModel();
+
+            b.add("$T.of(", ImmutableMap.class);
```

Comment:
should we use `builder()`, since `of` only supports up to 4 KV pairs?

Reply:
I tried to use that initially, but `ImmutableMap.builder().build()` returns `ImmutableMap<K, V>`, so its not compatible. We'd have to update `build()` to return `<K, V> ImmutableMap<K, V>` like `of()` does , which would entail an unchecked cast. WDYT?

```
        public <K, V> ImmutableMap<K, V> build() {
            HashMap<K, V> builtMap = (HashMap<K, V>) new HashMap<>(entries);
            return new ImmutableMap<>(builtMap);
        }
```

Reply:
Will merge as is, can update in follow up PR if needed, or can also add new `of()` overloads to support more pairs if necessary in future